src/appimagetool.c

```diff
@@ -890,10 +914,27 @@ main (int argc, char *argv[])
             exit(1);
         }
         
-        /* If the user has not provided update information but we know this is a Travis CI build,
-         * then fill in update information based on TRAVIS_REPO_SLUG */
+        /* If the user has not provided update information but we know this is a CI build,
+         * then fill in update information based on well-known CI environment variables */
         if(guess_update_information){
-            if(travis_repo_slug){
+
+            if(github_repository_name){
+                if(!github_token) {
+                    printf("Will not guess update information since $GITHUB_TOKEN is missing\n");
+                } else {
+                    gchar *zsyncmake_path = g_find_program_in_path ("zsyncmake");
+                    if(zsyncmake_path){
+                        char buf[1024];
+                        // gh-releases-zsync|probono|AppImages|latest|Subsurface-*x86_64.AppImage.zsync
+                        sprintf(buf, "gh-releases-zsync|%s|%s|latest|%s*-%s.AppImage.zsync", github_repository_owner, github_repository_name, app_name_for_filename, arch);
+                        updateinformation = buf;
+                        printf("Guessing update information based on $GITHUB_REPOSITORY=%s\n", github_repository);
+                        printf("%s\n", updateinformation);
+                    } else {
+                        printf("Will not guess update information since zsyncmake is missing\n");
+                    }
+                }
+            } else if(travis_repo_slug){
```

Comment:
Apparently https://www.travis-ci.com/ still exists, so we cannot rule out that some of our users might still be using it. I'd hence leave the code in for now but no longer document it. Maybe add a statement that this may be removed in the future, as soon as it would require active maintenance.

Reply:
No need to keep dead code IMO. And this code is dead. Everyone using Travis CI would punish themselves.

Reply:
Let's leave it in but undocumented for a while, maybe even print a warning that it may be removed in the future. But don't make breaking sudden changes right now, when the goal is to get people into switching from AppImageKit's appimagetool to this repo's one.